Homework3/task3.js

```diff
@@ -0,0 +1,8 @@
+for (let i =1; i <= 10; i++)
+console.log(`5 * ${i} = ${5*i} `);
```

Comment:
it would be better to wrap body of loop. Syntax you have used is valid but might be a bit confusing. 
```js
for (let i =1; i <= 10; i++) {
    console.log(`5 * ${i} = ${5 * i} `);
}
```